ticketbuyer/tb.go

```diff
@@ -277,8 +323,39 @@ func (tb *TB) buy(ctx context.Context, passphrase []byte, tip *wire.BlockHeader,
 		VSPFees:    poolFees,
 	})
 	if tix != nil {
-		for _, hash := range tix.TicketHashes {
+		for i, hash := range tix.TicketHashes {
 			log.Infof("Purchased ticket %v at stake difficulty %v", hash, sdiff)
+
+			if tb.cfg.VSP != nil {
+				credit := credits[i]
+				txHash := *hash
+				go func() {
+					time.Sleep(5 * time.Second)
+					tb.cfg.VSP.Queue(ctx, &txHash, credit)
+				}()
+			}
+		}
+		// unlock unused credits
+		if tb.cfg.VSP != nil {
```

Comment:
Not sure about the `if cfg.VSP != nil` here. If the user removes the VSP items from their config file (eg. they no longer want to use a VSP), the unlocking should still happen, otherwise these outputs will be locked indefinitely.